diagnostics_tools/CMakeLists.txt

```diff
@@ -114,6 +115,14 @@ install(DIRECTORY include/${PROJECT_NAME}/
 ## Testing ##
 #############
 
+roslint_cpp()
+
+roslint_python()
+
+if(CATKIN_ENABLE_TESTING)
+  roslint_add_test()
+endif()
+
 ## Add gtest based cpp test target and link libraries
```

Comment:
We should remove boilerplates and unused comments.

Reply:
Good point.

Reply:
See https://github.com/ChrisScianna/ROS-Underwater-RnD/pull/62/commits/9465ad229cddc6b8283c66632c2df4177247d350.